GymManager/GymManagerWebApp/FileReaders/JsonManager.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using GymManagerWebApp.Models;
+using Newtonsoft.Json;
+
+namespace GymManagerWebApp.FileReaders
+{
+    public class JsonManager
+    {   
+        public static string GetUsersFilePath()
+        {
+            string jsonPathInsideTheProject = "\\Databases\\Users.json";
+            string pathToProject = Path.GetDirectoryName(Path.GetDirectoryName(System.IO.Directory.GetCurrentDirectory()));
+            var fullPath = pathToProject + "\\GymManager\\GymManagerWebApp\\bin\\Debug\\netcoreapp3.1\\" + jsonPathInsideTheProject;
+            return fullPath;
+        }
+
+        public static async Task<IList<User>> GetUsersAsync()
+        {
+            var filePath = GetUsersFilePath();
+            var listOfUsers =  await Task.Run(()=>JsonConvert.DeserializeObject<List<User>>(File.ReadAllText(filePath)));
```

Comment:
here you can write return await Task.Run(()=>JsonConvert.DeserializeObject<List<User>>(File.ReadAllText(filePath))).toList();

Reply:
and remove listOfUsers

Reply:
Variable removed, 
PS. "ToList" was not necessary.